docs/sphinx/slurm-tasks.rst

```diff
@@ -37,7 +37,7 @@ There are two ways to specify which scripts are run in the batch job. By default
     - base.echo:
         msg: Hello world, from batch job!
 
-is run with ``se batch.yml``, a batch job will be queued, which eventually
+is run with ``se sbatch.yml``, a batch job will be queued, which eventually
```

Comment:
I think above (you have to expand the GitHub UI diff, or open the file) we say that the name is assumed to be `sbatch.yml`. Not sure if it should be `sbatch.yml` here, or `batch.yml` there :sweat_smile: